aten/src/ATen/native/native_functions.yaml

```diff
@@ -4696,14 +4696,14 @@
     QuantizedCPU: int_repr_quantized_cpu
     QuantizedCUDA: int_repr_quantized_cuda
 
-- func: _make_per_tensor_quantized_tensor(Tensor self, float scale, int zero_point) -> Tensor
+- func: _make_per_tensor_tensor_quantized_cpu(Tensor self, float scale, int zero_point) -> Tensor
   dispatch:
-    CPU: make_per_tensor_quantized_tensor_cpu
-    CUDA: make_per_tensor_quantized_tensor_cuda
+    CPU: make_per_tensor_tensor_cpu_quantized_cpu
+    CUDA: make_per_tensor_tensor_cuda_quantized_cpu
 
-- func: _make_per_channel_quantized_tensor(Tensor self, Tensor scale, Tensor zero_point, int axis) -> Tensor
+- func: _make_per_channel_tensor_quantized_cpu(Tensor self, Tensor scale, Tensor zero_point, int axis) -> Tensor
   dispatch:
-    CPU: make_per_channel_quantized_tensor_cpu
+    CPU: make_per_channel_tensor_cpu_quantized_cpu
```

Comment:
funnny one: cpu_quantized_cpu

Reply:
Thanks for pointing it out, @vadimkantorov. Will change and commit.